build_library/extra_sysexts.sh

```diff
@@ -2,4 +2,5 @@ EXTRA_SYSEXTS=(
   zfs:sys-fs/zfs
   podman:app-containers/podman,net-misc/passt
   python:dev-lang/python,dev-python/pip
+  vim:app-editors/vim
```

Comment:
I would like to avoid having too many extensions and treat it like packages. We can rather bundle similar things into a feature set, and give it an appropriate name (E.g., we wanted a dev extension with gcc, gdb and other stuff). Here we could call it `tools` and later on we also add an optional `extra-tools` extension for anything else we want to add that wasn't part of the base image (E.g., `htop`, `nano`?).